src/types/yargs.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Arguments, Options } from "yargs";
+
+interface Argument extends Arguments{
+    sourceMaps: boolean,
+    stripComments: boolean,
+    port: number,
+    dist: string,
+    src: string
+}
+
+interface Option extends Options{
```

Comment:
Where is this interface used?

Reply:
It is used in bundle.ts where we try to index the custom stripcomments option which does not exist in the yargs typing of options.